common/app/views/support/cleaner/VideoEmbedCleaner.scala

```diff
@@ -116,5 +121,30 @@ case class VideoEmbedCleaner(article: Article) extends HtmlCleaner {
     document
   }
 
+  private def wrapIframe(container: Element, iframe: Element) {
+    // Has no id to get data from capi so try and get from iframe
+    val videoWidth = iframe.attr("width")
+    val videoHeight = iframe.attr("height")
+    val hasWidth = videoWidth != "" && videoWidth != "0"
+    val hasHeight = videoHeight != "" && videoHeight != "0"
+
+    if (hasWidth && hasHeight) {
+      wrapCustom(container, videoWidth.toFloat, videoHeight.toFloat)
+    } else {
+      wrapHD(container)
+    }
+  }
+
+  private def wrapCustom(container: Element, width: Float, height: Float) {
+    val aspectRatio = width / height
+    val maxWidth =  Math.min(maxEmbedHeight * aspectRatio, width)
+    val paddingBottom = (1 / aspectRatio) * 100
+    container.wrap(s"""<div class="u-responsive-aligner" style="max-width: ${maxWidth}px;"><div class="embed-video-wrapper u-responsive-ratio" style="padding-bottom: ${paddingBottom}%;"></div></div>""")
```

Comment:
Could we add `px` to the `padding-bottom` inline style?

Reply:
Have you missed the "%" (it does look like a string interpolation char there!) or am is there something I'm missing?

Reply:
hah yes sorry completely missed it, and yep you're correct this should have been a %!